interface/modules/custom_modules/oe-module-weno/src/Services/TransmitProperties.php

```diff
@@ -129,37 +140,43 @@ private function getPatientInfo()
         //get patient data if in an encounter
         //Since the transmitproperties is called in the logproperties
         //need to check to see if in an encounter or not. Patient data is not required to view the Weno log
+        // TODO sjp To do prescriptions having to hava an active encounter isn't normal.
```

Comment:
From practical standp[oint, I am unable to do any orders (including prescriptions) in the larger EMRs I use unless I have an encounter opened. It is annoying, but think may be a billing or regulatory issue. Because of this, I frequently need to create a new "order" encounter whenever i need to make an order.

Reply:
A prescription isn't an order. The only reason for the encounter was Sherwin(according to Sherwin) used encounter to get user id(???)! I switch to use the returned users weno id.
I didn't see anywhere in weno portal where encounters are tracked.

Reply:
Just stating that in epic I can't do anything (including a rx) unless I have an active encounter open. It is super annoying and there ends up being a large amount of "order" type encounters in the chart.

But if not needed and can remove this requirement in weno code, then definitely better to remove the requirement.

Reply:
appreciate the clinical feedback.

Reply:
Wouldn't you think it's quite rare to have a prescription not related to an encounter which is why we have an encounter field in the prescription table?

Reply:
Sherwin added the field for weno according to him and looking through my clinic history very few encounters.
Also current prescription module doesn't require encounter. If they choose to open encounter it will still track but it is not required.